Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -26,9 +28,17 @@ class LoadDataFixturesDoctrineCommand extends DoctrineCommand
     /** @var SymfonyFixturesLoader */
     private $fixturesLoader;
 
-    public function __construct(SymfonyFixturesLoader $fixturesLoader)
+    public function __construct(SymfonyFixturesLoader $fixturesLoader, ?ManagerRegistry $doctrine = null)
     {
-        parent::__construct();
+        if ($doctrine === null) {
+            @trigger_error(sprintf(
+                'The "%s" constructor expects a "%s" instance as second argument, not passing it will throw a \TypeError in DoctrineFixturesBundle 4.0.',
+                static::class,
+                ManagerRegistry::class
+            ), E_USER_DEPRECATED);
+        }
+
+        parent::__construct($doctrine);
```

Comment:
Hello,

The Symfony `Command` constructor argument is waiting for a string and not an object. `https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Console/Command/Command.php#L69`

On previous SF versions (without PHP 7 type), it breaks here (https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Console/Command/Command.php#L660), which is calls by the `setName`.

Running `bin/console --env=test` for example returns
```
In Command.php line 659:
                                                                        
  Warning: preg_match() expects parameter 2 to be string, object given
```

Reply:
Please use 3.2.2 which fixes this. Thanks!